README.md

```diff
@@ -60,6 +60,10 @@ The following integrations are available and maintained by members of the Sentry
 
 - [Nette](https://github.com/Salamek/raven-nette)
 - [ZendFramework](https://github.com/facile-it/sentry-module)
+- [WordPress](https://wordpress.org/plugins/wp-sentry-integration/)
+- [Drupal](https://www.drupal.org/project/raven)
+- [OpenCart](https://github.com/BurdaPraha/oc_sentry)
+- ... feel free to be famous, create port to your favourite platform!
```

Comment:
Little typo (missing a, a).

"... feel free to be famous, create __a__ port to your favourite platform!"